app/controllers/events/routes.py

```diff
@@ -25,7 +29,9 @@ def events(term):
         listOfTerms = listOfTerms,
         rsvpedEventsID = rsvpedEventsID,
         currentTime = currentTime,
-        user = g.current_user)
+        user = g.current_user,
+        testing_currentTerm = g.current_term)
```

Comment:
do you need to pass this to the template?

Reply:
Yes, because there is an instance where we check if the user is a celtsAdmin or celtsStudentStaff to show the email button.

Reply:
removed line 33.